### PR TITLE
Introduce `jammy` as OS for workspace images

### PR DESCRIPTION
**Pull request by kylos101**
## Description

Introduce `jammy` as OS for workspace images

## Related Issue(s)

Fixes: https://github.com/gitpod-io/workspace-images/issues/1061

## How to test

Refer to `Options to consider` in the changelog.

## Release Notes

```release-note
Update OS to Jammy for Gitpod provided workspace images
```

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3544"></a>

### CODE REVIEW

**Comment by kylos101** [comment]
@atduarte could I ask for your help in editing/reviewing? :pray:

File: src/lib/contents/changelog/jammy.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: Workspace Images OS Update and Breaking Changes
+excerpt: Gitpod workspace images are being updated to use Ubuntu 22.04.2 LTS (Jammy Jellyfish). There are breaking changes to consider.
+date: 2023-03-27
+image: jammy.webp
+ogImage: jammy.webp
```
Comment (kylos101) [inline src/lib/contents/changelog/jammy.md:6]: @atduarte is there an image you'd like to provide in this changelog entry? :thinking: Or can we do without?

**Comment by Siddhant-K-code** [comment]
#3533 is merged  deployed now. We need to ship this ASAP to avoid confusions to users. 

:shipit:

**Comment by kylos101** [comment]
@atduarte @loujaybee are either of you able to help with a review, sharing an image, and Tweet?

**Comment by Siddhant-K-code** [comment]
I have updated the image & also added an instruction to file an issue if there are any problems in their workspaces in commit [`c8735f3`](https://github.com/gitpod-io/website/pull/3544/commits/c8735f37679eac7407034a3e2d3af2726a4f0322)

File: src/lib/contents/changelog/jammy.md (hunk 1 above)
Comment (Siddhant-K-code) [reply src/lib/contents/changelog/jammy.md:6]: resolved in https://github.com/gitpod-io/website/pull/3544#issuecomment-1486260079

**Review by Siddhant-K-code — APPROVED** [review]
Looks good to me ✔️ 

- For User: Instructions to file an issue, if they face any problem with their workspaces. ✅ 
- Alternative solutions to use tagged images. ✅ 
- Image asset ✅

File: src/lib/contents/changelog/jammy.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+## Breaking changes
+
+1. All maintained images have been updated from Focal to Jammy [[1](https://github.com/gitpod-io/workspace-images/#-images-well-maintain)][[2](https://github.com/gitpod-io/workspace-images/#-specific-images)] at [hub.docker.com/u/gitpod](https://hub.docker.com/u/gitpod).
+2. [`gitpod/workspace-ruby-2` is deprecated and will not be updated; Ruby 2.7 is EOL March 31, 2023](https://endoflife.date/ruby).
+3. Upgraded `gitpod/workspace-mongodb` to mongo v6. v5 doesn't support OpenSSL 3, [OpenSSL 3 comes out of the box with Jammy](https://packages.ubuntu.com/source/jammy/openssl).
+4. Updated `gitpod/workspace-node-lts` from Node 16 to Node 18, and also in `gitpod/workspace-full`.
+5. Updated `gitpod/workspace-node` from Node 18 to Node 19.
+6. Installed Python 3.9 in `gitpod/workspace-yugabytedb*`, changing its `base` ref from `full`. Refer to its [dockerfile](https://github.com/gitpod-io/workspace-images/blob/kylos101/jam/chunks/tool-yugabytedb/Dockerfile#L27) for background
+
+Additionally, `gitpod/workspace-node` and `gitpod/workspace-node-lts` workspace images will be upgraded to use Node 19.0.0 and Node 18.13.0, respectively.
```
Comment (atduarte) [inline src/lib/contents/changelog/jammy.md:33]: I think this is repeated.

```suggestion
```

**Review by atduarte — CHANGES_REQUESTED** [review]
Minor changes requested. Thanks a ton, @kylos101! Looks very good 🙏

File: src/lib/contents/changelog/jammy.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+If you'd like to continue using the current OS and Node versions, you may pin the workspace image version by changing the tag from `latest` to `2023-01-16-03-31-28` (e.g. `gitpod/workspace-node-lts:2023-01-16-03-31-28`) in your `.gitpod.yml` or `Dockerfile`.
```
[the remark this answers is not in the document]
Comment (kylos101) [reply src/lib/contents/changelog/jammy.md:35]: Great point, @atduarte , thank you.

**Review by axonasif — APPROVED** [review]
Looks perfect!

**Comment by kylos101** [comment]
Thanks for the suggestion @axonasif , accepted!

**Comment by kylos101** [comment]
@atduarte I could not merge this, it required your explicit approval (even though I accepted your changes). I dismissed your review (so that we can merge), and am sharing here for posterity.

**Comment by atduarte** [comment]
Sorry missed the above!

**Pull request merged**